tools/python_api/test/test_scan_polars.py

```diff
@@ -0,0 +1,10 @@
+import polars as pl
+from type_aliases import ConnDB
+
+def test_polars_basic(conn_db_readonly: ConnDB) -> None:
```

Comment:
Can you add a test where the df is not a valid polars or the user is trying to load from a polars which doesn't exist